askem.dbml

```diff
@@ -136,6 +136,7 @@ Table dataset {
   temporal_resolution varchar
   geospatial_resolution varchar
   annotations json
+  data_path varchar
```

Comment:
What's the difference between `url` and `data_path`?

Reply:
`url` is the original source and `data_path` is the where the file is currently stored. I was using `url` because sim runs original source was (kind of) on minio/s3. Now the concepts are cleanly separated again. `url` might be a link to the dataset on the CDC website and the `data_path` is the file we serve.

This does highlight an issue with the way we currently serve the dataset. We are including the `data_path` which might be information we don't want to expose to the API user. Then again, we can probably ignore it since Terarium and the sim service are the users